### DISCUSSION TITLE
Authentication failed(login with Google)

### DISCUSSION THREAD

**Discussion by MMana76** [outline/outline#3508] [Self Hosting]
**To Reproduce**
1. I installed outline on our Aws Ec2 instance
2. ps shows

```
docker-compose ps
       Name                     Command               State                    Ports                  
------------------------------------------------------------------------------------------------------
outline_outline_1    docker-entrypoint.sh sh -c ...   Up      0.0.0.0:3000->3000/tcp,:::3000->3000/tcp
outline_postgres_1   docker-entrypoint.sh postgres    Up      0.0.0.0:5432->5432/tcp,:::5432->5432/tcp
outline_redis_1      docker-entrypoint.sh redis ...   Up      0.0.0.0:6379->6379/tcp,:::6379->6379/tcp
```

I checked log
`{"error":"client_secret is missing.","stack":"TokenError: client_secret is missing.\n    at Strategy.OAuth2Strategy.parseErrorResponse (/opt/outline/node_modules/passport-oauth2/lib/strategy.js:373:12)\n    at Strategy.OAuth2Strategy._createOAuthError (/opt/outline/node_modules/passport-oauth2/lib/strategy.js:420:16)\n    at /opt/outline/node_modules/passport-oauth2/lib/strategy.js:177:45\n    at /opt/outline/node_modules/oauth/lib/oauth2.js:191:18\n    at passBackControl (/opt/outline/node_modules/oauth/lib/oauth2.js:132:9)\n    at IncomingMessage.<anonymous> (/opt/outline/node_modules/oauth/lib/oauth2.js:157:7)\n    at IncomingMessage.emit (node:events:538:35)\n    at IncomingMessage.emit (node:domain:475:12)\n    at endReadableNT (node:internal/streams/readable:1345:12)\n    at processTicksAndRejections (node:internal/process/task_queues:83:21)","level":"error","message":"Error during authentication"}`

Browser shows 302 redirect.
Why?

**Expected behavior**
We expect to login.

**Screenshots**

![Screenshot from 2022-05-09 10-05-15](https://user-images.githubusercontent.com/105203308/167376560-1aa8f979-ec70-4fc8-b0a6-2ca675897f15.png)

**Outline (please complete the following information):**
- Install: [getoutline.com or self hosted]
- Version: [commit sha if self hosted]

**Desktop (please complete the following information):**
 - Ubuntu
 - Chrome,Firefox

**Mobile (please complete the following information):**

**Comment by mrloldev** [comment]
could you get a solution?

**Comment by vishnuunni910** [comment]
Ran into same issue, were you able to find a solution?
